modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -208,7 +208,7 @@ The low watermark for available memory at process start.
 
 === redpanda_memory_free_memory
 
-Available memory in bytes.
+Total memory potentially available (includes both free and reclaimable memory) to a CPU shard (core), in bytes.
```

Comment:
I don't think this is correct.

`redpanda_memory_free_memory` = Unallocated memory

`redpanda_memory_available_memory` = Unallocated + reclaimable

The problem with the original wording for this line is that it uses "Available", which is confusing when we use that in the other metric name to indicate something different.